airbyte-integrations/connectors/source-twilio-singer/source_twilio_singer/spec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/twilio",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Source Twilio Api Singer Spec",
+    "type": "object",
+    "required": ["start_date", "account_sid", "auth_token", "user_agent"],
+    "additionalProperties": false,
+    "properties": {
+      "start_date": {
+        "type": "string",
+        "description": "The date from which you'd like to replicate data for Twilio API, in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated.",
+        "examples": ["2020-08-14T00:00:00Z"],
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+      },
+      "account_sid": {
+        "type": "string",
+        "description": "The Twilio account ID to use when pulling data from the Twilio API."
+      },
+      "auth_token": {
+        "type": "string",
+        "description": "The value of the access token generated. See the <a href=\"https://docs.airbyte.io/integrations/sources/twilio\">docs</a> for more information",
+        "airbyte_secret": true
+      },
+      "user_agent": {
+        "type": "string",
+        "description": "The API user email for Twilio account",
```

Comment:
Also we can display a "nice" title on the UI for this field by adding the "title" field. For example if we add

```
      "user_agent": {
        "title": "Twilio Email",
        "type": "string",
        ...
```
the UI will display this as "Twilio Email" instead of `user_agent` which is more readable.